apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -176,14 +180,32 @@ export class AppMessagesConverter {
 
 		const attachments = this._convertAppAttachments(message.attachments);
 
+		let _id = message.id;
+		let ts = message.createdAt;
+		let _updatedAt = message.updatedAt;
+
+		if (!isPartial) {
+			if (!message.id) {
+				_id = Random.id();
+			}
+
+			if (!message.createdAt) {
+				ts = new Date();
+			}
+
+			if (!message.updatedAt) {
+				_updatedAt = new Date();
+			}
```

Comment:
I don't think we need to pass this as the updateOne will override it anyways with a new _updatedAt.

Reply:
That's fair. I wanted to keep it in because the function did it previously. But the impact is minimal anyway.